src/app.jsx

```diff
@@ -116,17 +119,18 @@ export const Application = () => {
                     });
                 })).then(res => {
                     Promise.all(res.map(file => {
-                        return cockpit.spawn(["file", "/" + path.join("/") + "/" + file.path], { superuser: "try" }).then(res => {
+                        return cockpit.spawn(["file", currentPath + "/" + file.path], { superuser: "try" }).then(res => {
                             return { ...file, info: res.split(":")[1].slice(0, -1) };
                         });
                     })).then(res => setFiles(res));
                 });
             });
         };
         getFsList();
-    }, [path]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Please don't disable rules. this can be easily fixed by adding the currentPath in the dependency array.